pkg/release-controller/types.go

```diff
@@ -59,6 +59,9 @@ type Release struct {
 	Target *imagev1.ImageStream
 	// Config holds the release configuration parsed off of Source.
 	Config *ReleaseConfig
+
+	// Inconsistency holds the inconsistency annotations parsed of Source
+	Inconsistency Inconsistencies
```

Comment:
I am not really a fan of adding this at the release.  
Personally, for the UI elements, I would rely on something as simple as...
In `release.go`:
```
func (r *Release) HasInconsistencies() bool {
	for _, tag := range r.Source.Spec.Tags {
		if _, ok := tag.Annotations[ReleaseAnnotationInconsistency]; ok {
			return true
		}
	}
	return false
}
```
And then in `http.go`, just pass:
```
{{ tableLink $release.Config $tag $release.HasInconsistencies }}
```
Also, make sure to update **all** occurrences of `tableLink` I'm pretty sure you missed one. ;)